docs/ROADMAP.md

```diff
@@ -0,0 +1,112 @@
+## Q3 Roadmap - Tentative
+
+### Overview
+
+---
+
+In Q3, there are a few groups of tasks we'd like to work on:
+
+- **Continue with the stories from Q2**: Authentication, Authorization,
+  Inclusion of Related models
+- **Feature parity gap**: Declarative support
+- **Internal tooling/infrastructure**: support Node.js 12 in juggler &
+  connectors, reduce build time, etc.
+- **Juggler-next**: some groundwork to prepare us to work on juggler-next, e.g.
+  spike on how the code resides affect our build process
+
+### Scope
+
+---
+
+#### Authentication
+
+- [ ] Token based authentication in API Explorer.
+      [Spike](https://github.com/strongloop/loopback-next/issues/2027)
+- [ ] enable class level `@authenticate` decorator:
+      https://github.com/strongloop/loopback-next/issues/2460
+  - Question: should we include this in Q3?
+
+#### Authorization
+
+- [ ] Common layer for authentication & authorization
+      https://github.com/strongloop/loopback-next/issues/2900
+- [ ] Add authorization to example-shopping repo
+
+#### Inclusion of Related Models
+
+- awaiting to @bajtos' spike to create the tasks
+
+#### From model definition to REST API with no custom repository/controller classes (Declarative Support) https://github.com/strongloop/loopback-next/issues/2036
+
+- [ ] [CRUDRESTController](https://github.com/strongloop/loopback-next/issues/2736)
+- [ ] [KeyValueRESTController](https://github.com/strongloop/loopback-next/issues/2737)
+- [ ] [Spike: Booter for creating REST APIs from model files](https://github.com/strongloop/loopback-next/issues/2738)
+
+#### CI cleanup https://github.com/strongloop-internal/scrum-apex/issues/422
+
+- [ ] Fix CI for
+      [dashdb](https://github.com/strongloop/loopback-connector-dashdb/issues/76)
+      and [db2](https://github.com/strongloop/loopback-connector-db2/issues/130)
+      connector
+- [ ] Test juggler 3.x and 4.x on connectors
+
+#### Support of Node.js 12 https://github.com/strongloop/loopback-next/issues/3072
+
+- [ ] [SQL connectors](https://github.com/strongloop/loopback-next/issues/3110)
+- [ ] [noSQL connectors](https://github.com/strongloop/loopback-next/issues/3111)
+- [ ] [service connectors](https://github.com/strongloop/loopback-next/issues/3112)
+
+#### Internal tooling - to address build time
+
+_Note: I'm copying from the slack discussion:
+https://ibmapim.slack.com/archives/C0WCN2EBG/p1560265245146700_
+
+- Spike: Investigate how to use Azure Pipelines to replace AppVeyor
+- pick a single Node.js version to test on Windows and MacOS. Only on Linux test
+  all Node.js versions.
+
+#### Documentation improvement https://github.com/strongloop/loopback-next/pull/2925
+
+- _Need to create a more concrete plan and tasks._
+
+#### Miscellaneous
+
+- Review of
+  [DB2 connector on IBM i](https://github.com/strongloop/loopback-connector-ibmi)
+
```

Comment:
We have 9 bugs that are not labelled as "good first issue" and "help wanted". Let's try to fix as many of them as we can:

https://github.com/strongloop/loopback-next/issues?q=is%3Aopen+is%3Aissue+label%3Abug+-label%3A%22good+first+issue%22+-label%3A%22help+wanted%22

There are also 24 issues labelled as "developer experience", it would be great to implement few of them (e.g. 5-10).

https://github.com/strongloop/loopback-next/issues?utf8=✓&q=is%3Aopen+is%3Aissue+-label%3A%22good+first+issue%22+-label%3A%22help+wanted%22+label%3Adeveloper-experience+